suite-native/app/package.json

```diff
@@ -15,12 +15,19 @@
         "pods": "npx pod-install"
     },
     "dependencies": {
+        "@formatjs/intl-datetimeformat": "^6.2.0",
```

Comment:
Please use `yarn why` for all these dependencies and try to unify version. You used latest version but we already have some older version in `node_modules` because they are dependencies of `@formatjs`

Reply:
Looks good.

<img width="779" alt="Snímek obrazovky 2022-09-22 v 17 51 25" src="https://user-images.githubusercontent.com/2227592/191795606-0a239e59-11d3-4791-acb9-7b9cfc7b9b04.png">